package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "big-js": "^3.1.3",
```

Comment:
- [ ] Please don't forget to fill your MIT license file [link](https://github.com/Christabelle143/math-magicians/blob/calculator-events/MIT.md). You can remove square brackets from your name and write `web app` instead of `[APP TYPE]`, also change year to 2022